nodejs/week2/app.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express');
+const app = express();
+const documents = require ("./documents.json");
```

Comment:
This is fine, but try and use es modules :)

Reply:
Thanks for feedback i changed it now using import :)